docs/release/trg-8/trg-8-04.md

```diff
@@ -65,9 +66,9 @@ jobs:
           image-ref: "tractusx/<your-image>:<version>" # Pull image from Docker Hub and run Trivy vulnerability scanner
           format: "sarif"
           output: "trivy-results.sarif"
-          exit-code: "1" # Trivy exits with code 1 if vulnerabilities are found, causing the workflow step to fail.
           severity: "CRITICAL,HIGH" # While vulnerabilities of all severities are reported in the SARIF output, the exit code and workflow failure are triggered only by these specified severities (CRITICAL or HIGH).
-          hide-progress: false
```

Comment:
Please dont remove this. Let it stay after the line 
`severity: "CRITICAL,HIGH"`